File: frontend/src/app/application/service/notification/notification-dto.ts
export class NotificationDto {

  userId: number;
  content: string;
  relatedId: number;
  notificationType: string;

  constructor(userId: number, content: string, relatedId: number, notificationType: string) {
    this.userId = userId;
    this.content = content;
    this.relatedId = relatedId;
    this.notificationType = notificationType;
  }
}

File: mq-general/src/main/java/ripoff/facebook/amqp/NotificationDTO.java
package ripoff.facebook.amqp;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.io.Serializable;


@AllArgsConstructor
@NoArgsConstructor
@Getter
public class NotificationDTO implements Serializable {
    private Long userId;
    private String content;
    private Long relatedId;
    private NotificationType notificationType;
}
